srcpkgs/python3-telegram/patches/use-system-tdlib.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/telegram/tdjson.py b/telegram/tdjson.py
+index 091145e..8d3ad21 100644
+--- telegram/tdjson.py
++++ telegram/tdjson.py
+@@ -9,19 +9,10 @@ import pkg_resources
+ logger = logging.getLogger(__name__)
+ 
+ 
+-def _get_tdjson_lib_path() -> str:
+-    if platform.system().lower() == 'darwin':
+-        lib_name = 'darwin/libtdjson.dylib'
+-    else:
+-        lib_name = 'linux/libtdjson.so'
+-
+-    return pkg_resources.resource_filename('telegram', f'lib/{lib_name}')
+-
+-
+ class TDJson:
+     def __init__(self, library_path: Optional[str] = None, verbosity: int = 2) -> None:
+         if library_path is None:
+-            library_path = _get_tdjson_lib_path()
++            library_path = 'libtdjson.so.1.7.0'
```

Comment:
as asked earlier: can you make this forward-compatible? else this patch needs adaptation should the lib's version change and it might be missed